server/api.go

```diff
@@ -296,7 +296,7 @@ func (p *Plugin) getConnected(w http.ResponseWriter, r *http.Request) {
 		resp.GitHubClientID = config.GitHubOAuthClientID
 		resp.Settings = info.Settings
 
-		if info.Settings.DailyReminder && r.URL.Query().Get("reminder") == "true" {
+		if info.Settings.DailyReminder && r.URL.Query().Get("reminder") == "true" && p.HasNews(info) {
```

Comment:
Might be better to do this inside this if block after doing the time comparison since it will be a little heavy making network requests

Reply:
Will do.